src/tools/info/version_info.c

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (C) 2022, NVIDIA CORPORATION & AFFILIATES. ALL RIGHTS RESERVED.
+ * See file LICENSE for terms.
+ */
+
+#ifdef HAVE_CONFIG_H
+#  include "config.h"
+#endif
+#include "ucx_info.h"
+
+#include <ucs/sys/lib.h>
+#include <ucs/sys/string.h>
+
+#include <libgen.h>
```

Comment:
are these includes still needed?

Reply:
No, but I always thought that all courses shouldn't depend on the internal inclusions inside the headers. It may cause a problem if `ucs/sys/string.h` will stop using `libgen` for some reason. So I will better add `stdio.h` inclusion here then remove this one. WDYT?

Reply:
in UCX each header should be self-sufficient

Reply:
Yes, but we use `dirname` call directly which is not required to be provided from any of UCX header and that's we I didn't remove this include.

Reply:
see another comment: "better print full path, no need to remove file name"